sympy/core/basic.py

```diff
@@ -619,7 +619,7 @@ def canonical_variables(self):
             reps[b] = d
         return reps
 
-    def rcall(self, *args):
+    def rcall(self, *args, **kwargs):
```

Comment:
The addition of `kwargs` now serves no purpose in this PR.
I guess it can be either reverted or kept as feature, but if it is going to be added, it needs release note entry.

Reply:
Why would `rcall` have kwargs?

Reply:
I think I'll revert it in this PR.